src/mechanics/constitutive/inputoutput/EquivalentStrain.cpp

```diff
@@ -1,214 +0,0 @@
-/*
- * EquivalentStrain.cpp
```

Comment:
Shouldn't we keep thoose helper functions?

Reply:
now here https://github.com/nutofem/nuto/blob/PDE_remove_old/src/mechanics/constitutive/EngineeringStrainInvariants.h